Replace InfluxDB with Thanos on Cluster Sizing page

lucianvlad opened the pull request:
Replace InfluxDB with Thanos on Cluster Sizing page

cristiklein reviewed (approved):
Nice cleanup!

viktor-f commented:
While you are at it, I searched and found mentions of influxdb here as well:
https://elastisys.io/compliantkubernetes/operator-manual/credentials/#backup-and-long-term-logging-credentials
https://elastisys.io/compliantkubernetes/user-guide/metrics/#prometheus

Might be more as well

lucianvlad commented:
> While you are at it, I searched and found mentions of influxdb here as well: https://elastisys.io/compliantkubernetes/operator-manual/credentials/#backup-and-long-term-logging-credentials https://elastisys.io/compliantkubernetes/user-guide/metrics/#prometheus
> 
> Might be more as well

Replaced in all the places I could find and also updated the apps version for OVH to 0.27 (last known working version)

viktor-f commented on docs/operator-manual/ovh-managed-kubernetes.md:
```diff
@@ -106,7 +106,7 @@ EOF
 sops exec-file --no-fifo s3cfg.ini "s3cmd --config {} mb s3://${CK8S_ENVIRONMENT_NAME}-velero"
 sops exec-file --no-fifo s3cfg.ini "s3cmd --config {} mb s3://${CK8S_ENVIRONMENT_NAME}-es-backup"
-sops exec-file --no-fifo s3cfg.ini "s3cmd --config {} mb s3://${CK8S_ENVIRONMENT_NAME}-influxdb"
+sops exec-file --no-fifo s3cfg.ini "s3cmd --config {} mb s3://${CK8S_ENVIRONMENT_NAME}-thanos"
```
This whole page is deprecated so you don't need to update it.

viktor-f reviewed (approved):
Although if you want you can do the update in the table that I discussed in the other comment

viktor-f commented on docs/user-guide/metrics.md:
```diff
@@ -39,14 +39,14 @@ Prometheus and Grafana can help with this by making it easier to:
 
-| crd | apigroup | kind | used by | description |
+| crd | apigroup | kind | can be used by users |
 | :-- | :-- | :-- | :-- | :-- |
```
I think this is breaking the formating
```suggestion
| :-- | :-- | :-- | :-- |
```

lucianvlad replied:
missed it
changed it now
